### ISSUE TITLE
Add support for content resolvers

### ISSUE THREAD

**Issue by greenart7c3** [sebdeveloper6952/amberflutter#2]
When the user chooses the remember my choice option the app can call amber in background using content resolvers

You can look an example in flutter in this repo:

https://github.com/0xchat-app/nostr-dart

**Comment by sebdeveloper6952** [comment]
Thank you @greenart7c3 , I will take a look at this.

**Comment by greenart7c3** [comment]
can you send me a json os the sign event? it's problably some missing field

**Comment by sebdeveloper6952** [comment]
Sure thing, here you go @greenart7c3 :
```
{
  "pubkey":"a1f26e90ef29c877dd1820ac57cfa631efb21f864699c689be25643fd7325d7e",
  "kind":1,
  "content":"Hello from Amber Flutter!",
  "created_at":1702025528
}
```

**Comment by greenart7c3** [comment]
For me it worked with the latest example. Are you using the latest version of amber?

**Comment by sebdeveloper6952** [comment]
I am on amber v0.6.6. Okay thanks greenart, I will keep searching. It works for me using the default way of launching an intent. It's just this content resolver way that I am encountering the error.

**Comment by greenart7c3** [comment]
oh, i see now.
the content provider is not replacing the null fields
Try sending the tags and sig field in the json
tags can be am empty array and sig an empty string

**Comment by sebdeveloper6952** [comment]
thank you very much @greenart7c3, that was it!

**Comment by sebdeveloper6952** [comment]
closed with #3
